Documentation page (#8)
SundareshSankaran/py-sas-studio-custom-steps: README.md

# py-sas-studio-custom-steps: A Python package for SAS Studio Custom Steps

> This repository provides a Python class and associated methods to facilitate operations on SAS Studio Custom Steps.  [SAS Studio Custom Steps](https://go.documentation.sas.com/doc/en/sasstudiocdc/default/webeditorcdc/webeditorsteps/titlepage.htm) are low-code components designed for a variety of analytical and data engineering operations on the SAS Viya platform (commercially available software).  They help users create a user interface for easy provision of parameters, that can then be used to execute an associated SAS program.  Custom Steps provide easy access to common operations, boost code reusability, and help establish best practices in programming.

Author: ```suggestion
This repository provides a Python class and associated methods to facilitate operations on SAS Studio Custom Steps.  [SAS Studio Custom Steps](https://go.documentation.sas.com/doc/en/sasstudiocdc/default/webeditorcdc/webeditorsteps/titlepage.htm) are low-code components designed for a variety of analytical and data engineering operations on the SAS Viya platform (commercially available software).  They help users create a user interface for parameters, that can then be used to execute an associated SAS program.  Custom Steps provide easy access to common operations, boost code reusability, and help establish best practices in programming.
```

> As SAS Viya continues to welcome a diverse set of users with different preferences (e.g. programming vs. low-code / no-code), a Python-based framework for interacting with custom steps will aid code reusability, rapid development and testing of SAS solutions. This package can also be used to access and port custom step capabilities from non-SAS Studio applications, such as [SAS Viya Workbench](https://www.sas.com/en_us/software/viya/workbench.html).

Author: ```suggestion
As SAS Viya continues to welcome a diverse set of users with different preferences (e.g. programming, low-code / no-code), a Python-based framework for interacting with custom steps aids code reusability, rapid development and testing of SAS solutions. This package can also be used to access and port custom step capabilities from non-SAS Studio applications, such as [SAS Viya Workbench](https://www.sas.com/en_us/software/viya/workbench.html).
```

- *create_custom_step:* Write a custom step (CustomStep) object to a SAS Studio Custom Step (.step) file
- *list_keys:* List out all the attributes of a current custom step.

## Documentation

> Refer this page for a list of all available methods and attributes.

Author: ```suggestion
Refer this [page](./DOCUMENTATION.md) for a list of all available methods and attributes.
```
